src/test/java/programminglife/gui/GuiControllerTest.java

```diff
@@ -60,15 +57,15 @@ public void setUp() throws TimeoutException {
 
     @After
     public void tearDown() throws Exception {
-        DataManager.clearDB(TEST_DB);
-        DataManager.removeDB(TEST_DB2);
+        Cache.removeDB(TEST_DB);
+        Cache.removeDB(TEST_File);
         FxToolkit.cleanupApplication(this.pl);
     }
 
     @AfterClass
     public static void tearDownClass() throws Exception {
-        DataManager.removeDB(TEST_DB);
-        DataManager.removeDB(TEST_File);
+        Cache.removeDB(TEST_DB);
+        Cache.removeDB(TEST_File);
```

Comment:
This is not really a problem, and to not interfere with other branches, this should be fixed globally for all tests in a separate PR.